gameDataFetch.js

```diff
@@ -0,0 +1,34 @@
+const IgdbApiWrapper = require('./IgdbApiWrapper');
+
+const IgdbWrapper = new IgdbApiWrapper();
+
+function dataFetch(){
+
+};
+
+dataFetch.prototype.fillAllData = async function(data){
+    this.data = Object.assign({}, data);
+    await this.fillPlatformsData();
+    if(this.data.websites)
+        this.fillWebsitesCategoryData();
+    if(this.data.developers)
+        await this.fillDevelopersData();
+    this.fillReleaseDate();
+    return this.data;
+};
+
+dataFetch.prototype.fillPlatformsData = async function(){
+    this.data.platforms = await IgdbWrapper.getPlatforms(this.data.platforms); 
```

Comment:
This method will throw an error if you call it before fillAllData(). You should define this.data object in a constructor.
Also consider returning value here instead of setting this.data, in such case you can use these methods to fetch only platforms information from other methods.

Reply:
Apply it to all methods within this class.